.github/workflows/test.yml

```diff
@@ -17,6 +17,15 @@ jobs:
           python-version: 3.11
       - uses: pre-commit/action@v3.0.0
 
+      - name: remove homebrew
+        run: |
+          # This takes too long (over a minute), so we just rm -rf the important stuff
+          # brew list | xargs brew uninstall --force
```

Comment:
May want to document why we have to remove homebrew

Reply:
sure, will do in #163